doc/api/process.md

```diff
@@ -1044,6 +1044,34 @@ a code.
 Specifying a code to [`process.exit(code)`][`process.exit()`] will override any
 previous setting of `process.exitCode`.
 
+## process.getActiveHandles()
```

Comment:
I would really like to take the opportunity now to move these to the `async_hooks` module or `util` module and pull them off the `process` object.

Reply:
Why?

Reply:
Because I'd prefer not to expand the API surface area of `process` if we can reasonably avoid it.

Reply:
It's already there though and people use it. I don't see how it makes more sense to place it on the `async_hooks` or `util` modules when the data returned by these functions have to strictly do with what's going on in the process.

Reply:
Maybe you could discuss that in a future PR? I mean, I'm not qualified to answer about this, but I would agree that it's easier to keep the functions where they are.
On the other hand, they were not documented, and could be moved without any further notice